README.md

```diff
@@ -41,16 +41,16 @@ Phases:
 * Phase 2 release is extension packages, e.g. [{admiralonco}](https://pharmaverse.github.io/admiralonco/main/), [{admiralophtha}](https://pharmaverse.github.io/admiralophtha/main/), [{admiralvaccine}](https://pharmaverse.github.io/admiralvaccine/main/)
 
 
-|Release Schedule | Phase 1- Date and Packages                                        | Phase 2- Date and Packages                                                    |
-|---------------- | --------------------------                                        | --------------------------                                                    |   
-| Q3-2023         | September 4th                                                     | September 11th                                                                | 
-|                 | [{admiraldev}](https://pharmaverse.github.io/admiraldev/main/)    | [{admiralonco}](https://pharmaverse.github.io/admiralonco/main/)              | 
-|                 | {pharmaversesdtm}                                                    | [{admiralophtha}](https://pharmaverse.github.io/admiralophtha/main/)          | 
-|                 | [{admiral}](https://pharmaverse.github.io/admiral/main/)          | [{admiralvaccine}](https://pharmaverse.github.io/admiralvaccine/main/)        | 
-| Q4-2023         | December 4th                                                      | December 11th                                                                 | 
-|                 | [{admiraldev}](https://pharmaverse.github.io/admiraldev/main/)    | [{admiralonco}](https://pharmaverse.github.io/admiralonco/main/)              |
-|                 | {pharmaversesdtm}                                                    | [{admiralophtha}](https://pharmaverse.github.io/admiralophtha/main/)          |  
-|                 | [{admiral}](https://pharmaverse.github.io/admiral/main/)          | [{admiralvaccine}](https://pharmaverse.github.io/admiralvaccine/main/)        | 
+|Release Schedule | Phase 1- Date and Packages                                                  | Phase 2- Date and Packages                                                    | Phase 3- Date and Packages                                             |
+|---------------- | --------------------------                                                  | --------------------------                                                    | --------------------------                                             |
+| Q3-2023         | September 4th                                                               | September 11th                                                                | September 18th                                                         |
+|                 | [{pharmaversesdtm}](https://pharmaverse.github.io/pharmaversesdtm/main/)    | [{admiraldev}](https://pharmaverse.github.io/admiraldev/main/)                | [{admiralonco}](https://pharmaverse.github.io/admiralonco/main/)       |    
```

Comment:
We should change this to December release and add the next one for Q1 2024.  That way it isn't stale

Reply:
Hmm, but after december next release may be Q2 2024?

Reply:
We have a rule stated - releases are targeted for the first Monday of the last month of each quarter.

Does the Q1 2024 not align?

Reply:
Well it doesn’t if we move to a bi-yearly release schedule following 1.0. So maybe we can add something about that instead?

Reply:
I think then we should re-work this section stating our new goals and a new rule for a bi-annual release.  

Still include a blurb saying we have a release in Q4-2024 and shifting from 4 releases to 2 releases.  Next release is expected in June???

Also - can you link the pharmaversesdtm website?  I think that is a nice thing to show off!?!

Reply:
@manciniedoardo , @bms63 , do you think it is realistic that we release version 1.0 in Q4-2024?

I think at the moment we do not have a clear idea what we need to change for a stable version. (I have just started this week working on an overview.) Discussing, agreeing on, and implementing these changes this year seems quite challenging to me.

Thus I would just announce a Q1-2024 release.

Reply:
Please see discussion [here](https://github.com/pharmaverse/admiral/discussions/2089) around 1.0.